english/enjoy-DreamRide/jump-off/jump.md

```diff
@@ -0,0 +1,3 @@
+Jumping off the swing, fork in hand, you unexpectedly crash through the wall.
+
+You are in another room. This one is not made of marshmallows... it is made of lemon-lime scented shaving cream. It's a little hard to stand up in here.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.